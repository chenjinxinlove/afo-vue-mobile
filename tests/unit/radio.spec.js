import { mount } from '@vue/test-utils'
import Demo from '../../src/radio/demo'
import Vue from 'vue'
import Radio from '../../src/Radio'

describe('Rate.vue', () => {
  // 测试是否可以被使用
  it('use', () => {
    Vue.use(Radio)
    expect(Vue.component(Radio.name)).toHaveProperty('name', 'VueComponent')
  })
  // demo测试快照
  it('demo snapshot', () => {
    expect(mount(Demo)).toMatchSnapshot()
  })
})
