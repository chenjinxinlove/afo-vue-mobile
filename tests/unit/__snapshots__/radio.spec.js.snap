// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rate.vue demo snapshot 1`] = `
<div>
  <p>基础用法</p>
  <afo-radio-group>
    <div class="afo-radio"><span class="afo-radio__input"><input type="radio" class="afo-radio__control" value="1"> <afo-icon size="18px" type="round-border"></afo-icon></span> <span class="afo-radio__label afo-radio__label--false">单选框 1</span></div>
    <div class="afo-radio"><span class="afo-radio__input"><input type="radio" class="afo-radio__control" value="2"> <afo-icon size="18px" type="round-border"></afo-icon></span> <span class="afo-radio__label afo-radio__label--false">单选框 2</span></div>
  </afo-radio-group>
  <div style="height: 20px;"></div>
  <p>禁用状态</p>
  <afo-radio-group disabled="">
    <div class="afo-radio"><span class="afo-radio__input"><input type="radio" class="afo-radio__control" value="1"> <afo-icon size="18px" type="round-border"></afo-icon></span> <span class="afo-radio__label afo-radio__label--false">单选框 1</span></div>
    <div class="afo-radio"><span class="afo-radio__input"><input type="radio" class="afo-radio__control" value="2"> <afo-icon size="18px" type="round-border"></afo-icon></span> <span class="afo-radio__label afo-radio__label--false">单选框 2</span></div>
  </afo-radio-group>
</div>
`;
